lib/worker_threads.js

```diff
@@ -18,6 +19,7 @@ module.exports = {
   MessageChannel,
   moveMessagePortToContext,
   threadId,
+  shareEnv,
```

Comment:
Hm – I don’t think it’s quite comparable, because `fs.constants`, `os.constants`, `zlib.constants` etc. provide constant values from the native layer (mostly from the OS itself), so things where the value is important, whereas this one is about object identity? I think the closest things we’d have are `util.inspect.custom` and `util.promisify.custom`?

Reply:
I see. I did a rough search and looks like the public symbols exported are:

- `util.inspect.custom`
- `util.promisify.custom`
- `repl.REPL_MODE_SLOPPY`
- `repl.REPL_MODE_STRICT`

this one is probably closer to the repl ones. Do you think we should follow the first pattern or the second?

Reply:
I’d personally also be okay with `worker.SHARE_ENV`, if you think that that makes more sense?

Reply:
I think `worker.SHARE_ENV` is more consistent with our existing style (though it's not ideal that we have different styles for options and hooks :/)